presto-main/src/test/java/com/facebook/presto/execution/scheduler/TestPhasedExecutionSchedule.java

```diff
@@ -245,7 +248,7 @@ private static PlanFragment createFragment(PlanNode planNode)
             types.put(symbol, VARCHAR);
         }
         return new PlanFragment(
-                new PlanFragmentId(planNode.getId() + "_fragment_id"),
+                new PlanFragmentId(nextPlanFragmentId.incrementAndGet()),
```

Comment:
hmm... so the id in `PlanNode` doesn't necessarily match the stageId / fragmentId right?

Reply:
Never mind, the plan node id in this tests are just strings ... :)